AppCheckCore/Sources/Core/GACAppCheckLogger.m

```diff
@@ -44,11 +44,11 @@ + (void)setLogLevel:(GACAppCheckLogLevel)logLevel {
 
 #pragma mark - Helpers
 
-NSString *GACAppCheckMessageCodeEnumToString(GACAppCheckMessageCode code) {
+static NSString *MessageCodeEnumToString(GACAppCheckMessageCode code) {
   return [[NSString alloc] initWithFormat:@"I-GAC%06ld", (long)code];
 }
 
-NSString *GACAppCheckLoggerLevelEnumToString(GACAppCheckLogLevel logLevel) {
+NSString *LoggerLevelEnumToString(GACAppCheckLogLevel logLevel) {
```

Comment:
`static` here?

Reply:
Thanks! I reverted it from `static` to copy/paste the warning message and didn't change it back before committing 🤦‍♂️